src/Commands/StartRoadRunnerCommand.php

```diff
@@ -84,7 +84,7 @@ public function handle(ServerProcessInspector $inspector, ServerStateFile $serve
             '-o', 'http.pool.supervisor.exec_ttl='.$this->maxExecutionTime(),
             '-o', 'http.static.dir=public',
             '-o', 'http.middleware=static',
-            '-o', app()->environment('local') ? 'logs.mode=production' : 'logs.mode=none',
+            '-o', 'logs.mode=production',
```

Comment:
This makes sense, but it would also be possible to just [override the logging mode for the `server` channel](https://roadrunner.dev/docs/intro-config) by adding:
```php
'-o', 'logs.channels.server.mode=production',
```
From what I can tell, this would suppress other noisy messages in production. Interestingly though, the mode `none` is not even documented...

Reply:
Let's keep the "logs.mode=production" for now.